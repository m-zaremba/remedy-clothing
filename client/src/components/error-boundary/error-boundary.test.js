import React from 'react';
import { shallow } from 'enzyme';

import ErrorBoundary from './error-boundary.component.jsx';

describe('testing ErrorBoundary component', () => {
  it('should render ErrorBoundary component', () => {
    expect(shallow(<ErrorBoundary />)).toMatchSnapshot();
  });
});
