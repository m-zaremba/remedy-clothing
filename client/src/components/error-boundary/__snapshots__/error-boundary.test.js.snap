// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing ErrorBoundary component should render ErrorBoundary component 1`] = `""`;
